### PR TITLE
Fix some variable substitution issues;

### PR DESCRIPTION
**Pull request by ycliuhw**
Fix some variable substitution issues;

### CODE REVIEW

File: snap/local/wrappers/fetch-oci [hunk 1]
```diff
@@ -29,19 +29,15 @@ fi
...
 echo "Wait for microk8s to be ready if needed."
-wait_result=$(microk8s.status --wait-ready --timeout 30 2>&1)
-if [ $? -ne 0 ]; then
-    echo "${wait_result}"
-    exit 0
-fi
-
-juju_version=$(/snap/bin/juju version | rev | cut -d- -f3- | rev)
-oci_image="docker.io/jujusolutions/jujud-operator:$juju_version"
+microk8s.status --wait-ready --timeout 30 2>&1
```
Comment (wallyworld) [inline snap/local/wrappers/fetch-oci:32]: we don't need the result check?

**Review by wallyworld — APPROVED** [review]

**Comment by ycliuhw** [comment]
["connect-plug-peers": error: cannot communicate with server](https://trello.com/c/hYFGtbIG/1609-connect-plug-peers-error-cannot-communicate-with-server)

File: snap/local/wrappers/fetch-oci (hunk 1 above)
Comment (ycliuhw) [reply snap/local/wrappers/fetch-oci:32]: nah, the internal exit code won't be passed out.

**Comment by ycliuhw** [comment]
$$merge$$

**Pull request merged**